raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/TraceEmitPunctuator.java

```diff
@@ -175,6 +177,25 @@ public void punctuate(long timestamp) {
             rawSpanList.size());
       }
 
+      // report entries in spanStore
+      if (spanStoreCountRateLimiter.tryAcquire()) {
+        tenantToSpanStoreCountCounter
+            .computeIfAbsent(
+                tenantId,
+                k ->
+                    PlatformMetricsRegistry.registerCounter(
+                        SPAN_STORE_COUNT, Map.of("tenantId", k)))
+            .increment(spanStore.approximateNumEntries() * 1.0);
+      }
+
+      // report count of spanIds per trace
+      tenantToSpanPerTraceCounter
+          .computeIfAbsent(
+              tenantId,
+              k -> PlatformMetricsRegistry.registerCounter(SPANS_PER_TRACE, Map.of("tenantId", k)))
+          .increment(spanIds.size() * 1.0);
```

Comment:
Just curious: why do we multiple with `* 1.0`?

Reply:
`double` param is expected